src/containers/HeatMap/heatMap.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Icon } from 'antd';
+import ReactTooltip from 'react-tooltip';
+import CalenderHeatMap from 'react-calendar-heatmap';
+import moment from 'moment';
+
+import LogoHeader from '../../components/LogoHeader';
+import JournalCard from '../../components/JournalCard';
+import NavigationBar from '../../components/navigationBar';
+import { ReactComponent as NoJournals } from '../assets/images/noJournals.svg';
+import 'react-calendar-heatmap/dist/styles.css';
+import './heatMap.css';
+
+const heatMap = props => {
+  const { data, journals, handleClick } = props;
+
+  const toolTipData = value => {
```

Comment:
@yosefalnajjarofficial I think you should move this function out of the heatMap component's render function and up to the top of the file -
Any function defined inside a component's render function / inside a functional component (like this) is a little bit inefficient in terms of memory, because whenever React re-renders this function, you're creating a new copy of the same function in memory.  It's best to try to move functions like that out of render so that you don't recreate the same function over and over every time a component renders.